### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by Harsha-Ambati**

### CODE REVIEW

File: firstpr/harsha.md [hunk 1]
```diff
@@ -1 +1,2 @@
 ##### Harsha Vardhan from india . B.tech student
```
Comment (arthurbuhl) [inline firstpr/harsha.md:1]: Please always use capital `I` for `India`, just like you have done it for your first and last name.

**Comment by UnevenCoder** [comment]
Can you commit again and see  :)

**Comment by Harsha-Ambati** [comment]
ok! sure

**Comment by Harsha-Ambati** [comment]
i have made changes already and my merge was done!! into main repo..you can close this one

**Comment by piratelicorne** [comment]
I saw the changes were merged in #135

**Review by piratelicorne — APPROVED** [review]
I saw the changes were merged in the PR #135

**Comment by johnjacobkenny** [comment]
@piratelicorne great spot! 🙂

**Pull request closed without merging**